devicon.json

```
@@ -7945,6 +7945,39 @@
         "color": "#bad05e",
         "aliases": []
     },
+    {
+        "name": "v8",
+	    "altnames": [
+            "v8 engine"
+        ],
+        "tags": [
+            "javascript-runtime",
+            "framework",
+            "javascript",
+            "web",
+            "cpp"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#00C4CC",
+	    "aliases": [
+            {
+                "base": "original",
+                "alias": "original-wordmark"
+            },
+            {
+                "base": "plain",
+                "alias": "plain-wordmark"
+            },
```

Removing trailing comma. 🙂
```suggestion
            }
```